templates/imagehub-gateway.yaml

```diff
@@ -44,11 +44,20 @@ metadata:
   name: imagehub-web
 spec:
   hosts:
-  - "imagehub.meshery.io"
```

Comment:
@pranav-bhatt this will work, although, in context of the larger set of istio workshops that we give, I wonder if keeping imagehub.meshery.io and and requirement for the user to create a local /etc/hosts entry is the way to go.  // @kumarabd

Reply:
If you see the imagehub PR ive created, there is a command ive added which gives you the exact ip+port

Reply:
@kumarabd as of now suggests to keep this in, due to other sample apps clashing ig